src/layers/domain/core/product_team/v3.py

```diff
@@ -0,0 +1,56 @@
+from uuid import UUID
+
+from attr import dataclass
+from domain.core.aggregate_root import AggregateRoot
+from domain.core.cpm_product import CpmProduct, CpmProductCreatedEvent
+from domain.core.device.v2 import Device, DeviceCreatedEvent, DeviceType
+from domain.core.enum import Status
+from domain.core.event import Event
+from domain.core.validation import ENTITY_NAME_REGEX
+from pydantic import Field
+
+
+@dataclass(kw_only=True, slots=True)
+class ProductTeamCreatedEvent(Event):
+    id: UUID
+    name: str
+    ods_code: str
+
+
+class ProductTeam(AggregateRoot):
+    """
+    A ProductTeam is the entity that owns Products, and is derived from ODS
+    Organisations.  A single ODS Organisation can be mapped onto multiple
+    ProductTeams, meaning that `ods_code` is not unique amongst ProductTeams.
+    """
+
+    id: UUID
+    name: str = Field(regex=ENTITY_NAME_REGEX)
+    ods_code: str
+
+    def create_device(
```

Comment:
We don't need the ability to create a device here? As Product Teams should only be able to create Cpm Products

Reply:
I think we said that we wouldnt go too deep into future work - cause it tends to then lead down a rabbit hole, we can have a user story (or a follow up task) that reads along the lines of "I can only create CPM products as a product team" or something like that 

You are correct though, we just need to avoid rabbit holes where possible i think

Reply:
I think Megan's point in this case makes sense though as it's a new version.